README.md

```diff
@@ -80,9 +80,9 @@ class Main{
 ### Execute payment
 
 ```java
-class Main{
-    public static void main(){
-        PaymentService service = new PaymentSerice();
+class Main {
+    public static void main(String[] args) {
+        PaymentService service = new PaymentService();
         service.setup("localhost", 10012, "user", "password");
         
         PaymentInstruction paymentInstruction = new PaymentInstruction(
```

Comment:
Since PR #12 we have additional "corda" folder in the package names for the classes. Can you, please, update imports as well to support the new layout?
`com.gevamu.corda.flows.PaymentInstruction`, `com.gevamu.corda.states.<...>`